doc/usecases/polyculture/map_specific_favorite_groups.md

```diff
@@ -0,0 +1,24 @@
+# Use Case: Map Specific Favorite Groups
+
+## Summary
+
+- **Scope:** Plants Layer
+- **Level:** User Goal
+- **Actors:** App User
+- **Brief:** The user change a set of favorite plant groups per map.
+- **Status:** Draft
+
+## Scenarios
+
+- **Precondition:**
+  - The user is logged in to the app.
+  - The plants layer is selected.
+  - At least one plant is placed
+- **Main success scenario:**
+  - The user adds or removes a plant group from her list of favorites.
+- **Alternative scenario:**
+- **Error scenario:**
```

Comment:
You should consider listing possible error scenarios that may occur. What I thought of for example:
 - The user attempts to add a plant group that is already in their favorites.
 - The user tries to remove a plant group that is not in their favorites.

Reply:
@NurramoX thank you for the review! Great suggestion! I agree that double-adding shouldn't work.

But: How can one remove something that is not in the list?

Reply:
magine a user has two instances of the application open in separate tabs, and coincidentally, they are on the same page in both tabs where group deletion is possible. In the first tab, the user can delete their favorite group, complete some tasks, close the tab, and then switch to the second tab. If the application does not have a hot-reloading feature, the favorite group will still appear in the second tab despite being deleted in the first one.

Sorry for the confusion

Reply:
Good point!